metadata-ingestion/docs/sources/hex/README.md

```diff
@@ -20,4 +20,12 @@ Currently, the [Hex API](https://learn.hex.tech/docs/api/api-reference) has some
 
 2. **Metadata Access**: There is no direct method to retrieve metadata for Collections, Status, or Categories. This information is only available indirectly through references within Projects and Components.
 
-Please keep these limitations in mind when working with the Hex connector.
\ No newline at end of file
+Please keep these limitations in mind when working with the Hex connector.
+
+For the Dataset - Hex Project lineage, the connector relies on the 
+[_Hex query metadata_](https://learn.hex.tech/docs/explore-data/cells/sql-cells/sql-cells-introduction#query-metadata) feature.
+Therefore, in order to extract lineage information, the required setup must include:
+
+- A separated warehouse ingestor (_eg_ BigQuery, Snowflake, Redshift, ...) with `use_queries_v2` enabled in order to fetch Queries.
+  This will ingest the queries into DataHub as `Query` entities and the ones triggered by Hex will include the corresponding _Hex query metadata_.
+- A DataHub server with version >= `0.3.10` so the `Query` entities are properly indexed by source (Hex in this case) and so fetched and processed by the Hex ingestor in order to emit the Dataset - Project lineage.
```

Comment:
We should include both the required oss and saas versions here

Reply:
addressed in commit 51c65f1a0a8fc0b2d8178c328f876623d9142098